.env.example

```
@@ -1,18 +1,18 @@
 HOST=0.0.0.0
-NODE_ENV=development
 LOG_LEVEL=info
 
-# Obsidian Vault
+# Obsidian Vault Configuration
 OBSIDIAN_VAULT_PATH=/path/to/your/obsidian/vault
 
 # ChromaDB Configuration
 CHROMADB_HOST=localhost
 CHROMADB_PORT=8000
 CHROMADB_COLLECTION_NAME=obsidian_embeddings
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CHROMADB_COLLECTION_NAME key should go before the CHROMADB_HOST key

(UnorderedKey)